lib/Extension/LanguageServerPhpCsFixer/LanguageServerPhpCsFixerExtension.php

```diff
@@ -31,10 +32,16 @@ public function load(ContainerBuilder $container): void
             function (Container $container) {
                 $path = $container->get(FilePathResolverExtension::SERVICE_FILE_PATH_RESOLVER)->resolve($container->parameter(self::PARAM_PHP_CS_FIXER_BIN)->string());
```

Comment:
you can use `$container->expect(FilePathResolverExtension::SERVICE_FILE_PATH_RESOLVER, PathResolver::class)` to avoid the phpstan error
`